### ISSUE TITLE
🐞💣 [Bug] Major disconnect when sending

### ISSUE THREAD

**Issue by kcw-grunt** [litecoin-foundation/litewallet-android#18]
# Overview
When sending the app fails showing a Send Failed . disconnect  This was a major disruption to services.  Users could not send and the support team were swamped.

##  Environment
- v2.4.2
- v2.5.0
- v2.6.0

## Screenshots
|Sample|
|-------|
|<img width="1576" alt="Screen Shot 2022-05-02 at 7 03 57 PM (1)" src="https://user-images.githubusercontent.com/2899463/166430627-402d24c6-45dd-48bf-bcbf-0d600b417859.png">|
 

## Resolution
It turns out the Android still relied on the api.loafwallet.org server which was recently shutdown.  In iOS . there is no server call to get LTC network fees.  The object is relatively stable and as you can see in Android calling a server for this can cause a major issue.  So, iOS relies on static values. And that is the resolution here.

Object:
Patch in :  `BRApiManager`
 ```
       //Operationally, it makes more sense to review the fees than rely on a server.
        // Especially, when the missing server causes a major disconnect
        // Using this hard code for a bi-yearly review. This matches the architecture in iOS
        // KCW: May 3, 2022
        String jsonString = "{'fee_per_kb': 10000, 'fee_per_kb_economy': 2500, 'fee_per_kb_luxury': 66746}";
```

**Comment by kcw-grunt** [comment]
cc: @vsima @RyanPetter

**Comment by kcw-grunt** [comment]
resolved in v2.6.1
